pages/studentAssessmentInstanceHomework/studentAssessmentInstanceHomework.ejs

```diff
@@ -66,22 +66,22 @@
           <thead>
             <% if (has_manual_grading_question && has_auto_grading_question) { %>
             <tr>
-              <th rowspan="2">Question</th>
-              <th class="text-center" colspan="3">Auto-grading</th>
-              <th class="text-center" rowspan="2">Manual grading points</th>
-              <th class="text-center" rowspan="2">Total points</th>
+              <th>Question</th>
+              <th class="text-center">Value</th>
+              <th class="text-center">Previous variants</th>
+              <th class="text-center">Auto-grading points</th>
+              <th class="text-center">Manual grading points</th>
+              <th class="text-center">Total points</th>
```

Comment:
Why don't we need the `colspan` and `rowspan` attributes anymore? Or said another way, why did we used to need them?

Reply:
This no longer seems to be relevant in the current diff.